plugins/filter_aws/aws.c

```diff
@@ -370,6 +400,264 @@ static int get_vpc_metadata(struct flb_filter_aws *ctx)
     return ret;
 }
 
+static void get_cluster_from_environment(struct flb_filter_aws *ctx)
+{
+    if(ctx->cluster == NULL) {
+        char* cluster_name = getenv("CLUSTER_NAME");
+        if(cluster_name) {
+            ctx->cluster = strdup(cluster_name);
```

Comment:
nit:
```suggestion
            ctx->cluster = flb_strdup(cluster_name);
```

Both code are doing same thing, just trivial code schematics suggestions to follow FluentBit patterns.

Reply:
Will update in revision